cmd/frcli/csv.go

```diff
@@ -10,10 +14,10 @@ import (
 // CSVHeaders returns the headers used for harmony csv records.
 var CSVHeaders = "Timestamp,OnChain,Type,Category,Amount(Msat),Amount(Fiat),TxID,Reference,BTCPrice,BTCTimestamp,FiatCurrency,Note"
 
-// csv returns a csv string of the values contained in a rpc entry. For ease
+// writeToCSV returns a csv string of the values contained in a rpc entry. For ease
 // of use, the credit field is used to set a negative sign (-) on the amount
 // of an entry when it decreases our balance (credit=false).
-func csv(e *frdrpc.ReportEntry) string {
+func writeToCSV(e *frdrpc.ReportEntry) string {
```

Comment:
is this rename fine? 😅  only did it to that i could import the csv package without renaming the import. But can defs just rename the import if that would be better

Reply:
yeah sure, not the finest choice of name if the first place! would separate it out into another commit, because I'm fussy like that, but def not required.